TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -1,23 +1,40 @@
 version: '3.7'
 services:
 
-  captureproxy:
+
+  # Run combined instance of Capture Proxy and Elasticsearch
+  capture_proxy_es:
     image: 'migrations/capture_proxy:latest'
     networks:
       - migrations
     ports:
       - "9200:9200"
-    command: /runJavaWithClasspath.sh org.opensearch.migrations.trafficcapture.proxyserver.Main  --kafkaConnection kafka:9092 --destinationUri  https://elasticsearch:9200  --insecureDestination --listenPort 9200 --sslConfigFile /usr/share/elasticsearch/config/proxy_tls.yml
+      - "19200:19200"
```

Comment:
I don't think that there's a reason to expose this port.  Sure, it could be valuable to us, but it's more secure, potentially less confusing, and more consistent to what was happening before if it the same ports remain exposed.